main.js

```diff
@@ -252,7 +252,7 @@ class DataEntryInstance extends InstanceBase {
 				default: '*',
 				useVariables: true,
 				isVisible: (opt) => {
-					return opt.formattype.startsWith('ecma')
+					return !!opt && typeof opt.formattype === 'string' && opt.formattype.startsWith('ecma')
```

Comment:
If the PR aims to make changes to `isVisible`, which is deprecated, why not just spend the time now to use `isVisibleExpression` instead?